go.mod

```diff
@@ -24,12 +24,13 @@ require (
 	github.com/openshift/machine-config-operator v0.0.1-0.20230724174830-7b54f1dcce4e
 	github.com/sergi/go-diff v1.1.0
 	github.com/stretchr/testify v1.9.0
-	golang.org/x/sync v0.8.0
+	golang.org/x/sync v0.10.0
```

Comment:
is this a direct artifact from the bump?

Reply:
All the changes here are derived by the bump of `golang.org/x/net -> v0.33.0` 
and calling `go mod tidy && go mod vendor` after that